ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/model/TopicNode.java

```diff
@@ -39,20 +38,26 @@
   private static final ObjectMapper OBJECT_MAPPER = TestJsonMapper.INSTANCE.get();
 
   private final String name;
+  private final JsonNode keySchema;
   private final JsonNode schema;
   private final int numPartitions;
   private final int replicas;
+  private final String keyFormat;
   private final String format;
 
   public TopicNode(
       @JsonProperty("name") final String name,
+      @JsonProperty("keySchema") final JsonNode keySchema,
       @JsonProperty("schema") final JsonNode schema,
```

Comment:
Can we rename `schema` and `format` to `valueSchema` and `valueFormat`? I realize there's a little extra effort needed to maintain backwards compatibility for users of the testing tool but I think not doing the rename will be confusing down the line, especially since `format` is used throughout the rest of the codebase to mean both key and value format (e.g., `CREATE STREAM ... WITH (FORMAT='JSON', ...);` is valid now).

Reply:
sure, I'll do this in a follow-up PR since it requires changing tons of test files